### PR TITLE
Units for Sleepers

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- Unit (seconds) field added to input#2 and output#2
- Removed the ```...``` from the [PROGRESS] message

## Related issue number
related to https://github.com/ITISFoundation/osparc-simcore/issues/2091

## Checklist

- [ ] I think the code is well written
- [ ] I updated the table of contents (make toc)
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

**Review by GitHK — COMMENTED** [review]
I see that you have altered the schema and the validator dose not recognise it.

Maybe @pcrespov @sanderegg have a better idea on how to handle this?

File: .github/workflows/sleeper.yml [hunk 1]
```diff
@@ -28,24 +28,24 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
-      - name: setup python environment
-        uses: actions/setup-python@v1.1.1
+      - name: setup python environmentactions/setup-python@v2
```
Comment (sanderegg) [inline .github/workflows/sleeper.yml:31]: revert that line? find/replace that went wrong?

**Review by sanderegg — APPROVED** [review]
this looks good. do you think you could add some of [sleeper issues](https://github.com/ITISFoundation/osparc-simcore/issues/2091) ? maybe the progress part ?

@GitHK I don't understand your comment with the schema change... this goes together with PR[#2145](https://github.com/ITISFoundation/osparc-simcore/issues/2145)

**Comment by odeimaiz** [comment]
> I see that you have altered the schema and the validator dose not recognise it.
> 
> Maybe @pcrespov @sanderegg have a better idea on how to handle this?

The schema will be updated [here](https://github.com/ITISFoundation/osparc-simcore/pull/2145) and then the validator will go through

File: .github/workflows/sleeper.yml (hunk 1 above)
Comment (odeimaiz) [reply .github/workflows/sleeper.yml:31]: It also looked weird to me, I took it from the jupyters one. Reverted, thanks.

**Review by pcrespov — APPROVED** [review]
schema in on its way

**Pull request merged**